### ISSUE TITLE
keep getting put into #focus mode across platforms

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#48917]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 9.0.31-20
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @mallenexpensify 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1725929489084449

## Action Performed:

1. Select preference mode as "Most recent"
2. Signed out of Desktop/Web
3. Signed in

## Expected Result:
User remains in Most recent mode

## Actual Result:
User forced into #focus mode

## Workaround:
Unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/7ce29dc8-396a-4715-b8cd-0fb87dd364a2

Add any screenshot/video evidence

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by mallenexpensify** [comment]
This is also happening to @rushatgabhane who commented on the Slack thread
> backend bug. it keeps sending the nvp as false

The setting to mute sounds is also reset. Both have happened to me many times

**Comment by rushatgabhane** [comment]
internal issue. 

🎀 👀 🎀

**Comment by mallenexpensify** [comment]
Sorry for fussing with labels and assignment. I thought it might be helpful to try to get a different engineer assigned, but that shouldn't be necessary.

**Comment by puneetlath** [comment]
@mallenexpensify I'm headed OOO for a bit and I have too many NewDot Quality issues assigned right now. I'm going to use the auto-assigner to get someone else.

**Comment by MonilBhavsar** [comment]
I'll take a look today

**Comment by MonilBhavsar** [comment]
Submitted PR for review

**Comment by mallenexpensify** [comment]
@MonilBhavsar @rushatgabhane will this also fix the similar bug where `Mute all sounds from Expensify` is also toggled at the same time as being forced into @focus mode?

**Comment by rushatgabhane** [comment]
@mallenexpensify see https://expensify.slack.com/archives/C049HHMV9SM/p1727294519198659?thread_ts=1725929489.084449&cid=C049HHMV9SM

**Comment by mallenexpensify** [comment]
Commented [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1727479024498799?thread_ts=1727295024.816919&cid=C01GTK53T8Q), let's fix the problem of a system setting (mute sound) not being saved when signing out/in.  I'm guessing we want a stand-alone issue for that, right?

**Comment by MonilBhavsar** [comment]
Yes, that's a completely different issue. Happy to be assigned to it though!

**Comment by mallenexpensify** [comment]
Thx @MonilBhavsar , created an issues, assigned you and Rushat for the time being
- https://github.com/Expensify/App/issues/49952

**Comment by mallenexpensify** [comment]
Contributor+: @rushatgabhane due $250 via NewDot 

@rushatgabhane plz complete the BZ checklist. 
**BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:

- [ ] [@] The PR that introduced the bug has been identified. Link to the PR:
- [ ] [@] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:
- [ ] [@] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
- [ ] [@] Determine if we should create a regression test for this bug.
- [ ] [@] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
- [ ] [@BZ] Link the GH issue for creating/updating the regression test once above steps have been agreed upon:

**Comment by mallenexpensify** [comment]
Also confirming that after signing out/in on the latest staging version of Desktop, I was **not** forced into #focus mode. "Mute all sounds from Expensify" was toggled though, hahahhah (that's being addressed in a separate issue... somewhere)
<img width="724" alt="image" src="https://github.com/user-attachments/assets/c5cfbe01-a27c-47a0-92d1-a58e1654635c">

**Comment by mallenexpensify** [comment]
Thx @rushatgabhane , we should be all good here. 
Test case
- https://github.com/Expensify/Expensify/issues/434897
